From: Carlos Maddela <e7appew@gmail.com>
Date: Sat, 1 Dec 2018 03:25:18 +1100
Subject: Fix desktop file to use correct icon and keywords.

Description: Fix desktop file to use correct icon and keywords.
Author: Carlos Maddela <e7appew@gmail.com>
Forwarded: https://gitlab.gnome.org/GNOME/filemanager-actions/merge_requests/2
Last-Update: 2018-12-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 src/ui/fma-config-tool.desktop.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/ui/fma-config-tool.desktop.in b/src/ui/fma-config-tool.desktop.in
index ee0b7c3..b66713d 100644
--- a/src/ui/fma-config-tool.desktop.in
+++ b/src/ui/fma-config-tool.desktop.in
@@ -4,6 +4,7 @@ Type=Application
 _Name=FileManager-Actions Configuration Tool
 _Comment=Configure items to be added to the File-Manager context menus
 Exec=fma-config-tool
-Icon=fma-config-tool
+Icon=filemanager-actions
 Categories=System;FileTools;FileManager;
+Keywords=file tools;file manager;custom menu;custom action;
 StartupNotify=true
